app/controllers/Whereuat.scala

```diff
@@ -119,7 +105,7 @@ class Whereuat extends Controller {
   // account. This request should receive the client's properly formatted phone
   // number, GCM ID, and verification code in the request body, make sure the
   // verification code matches the one stored in the verifiers collection, and
-  // then add them to the clients collection if it does.
+  // then add them to the client collection if it does.
```

Comment:
For the future, don't change random stuff if it doesn't have anything to with the CL.